cognite_toolkit/_cdf_tk/loaders/_resource_loaders/extraction_pipeline_loaders.py

```diff
@@ -249,6 +249,7 @@ class ExtractionPipelineConfigLoader(
     kind = "Config"
     dependencies = frozenset({ExtractionPipelineLoader})
     _doc_url = "Extraction-Pipelines-Config/operation/createExtPipeConfig"
+    parent_resource = frozenset({ExtractionPipelineLoader})
```

Comment:
Bug in the ExtractionPipelineConfigLoader which is only relevant when the dump is implemented.